src/sql/azdata.d.ts

```diff
@@ -429,6 +429,23 @@ declare module 'azdata' {
 		options: { [key: string]: any };
 	}
 
+	/**
+	 * Information about available engine editions
+	 */
```

Comment:
I think this would be better as

`The possible values of the server engine edition`